packages/x-components/src/x-modules/queries-preview/store/module.ts

```diff
@@ -26,34 +26,37 @@ export const queriesPreviewXStoreModule: QueriesPreviewXStoreModule = {
   }),
   getters: { loadedQueriesPreview },
   mutations: {
-    clearQueryPreview(state, query) {
-      Vue.delete(state.queriesPreview, query);
+    clearQueryPreview(state, queryPreviewHash) {
+      Vue.delete(state.queriesPreview, queryPreviewHash);
     },
     setParams(state, params) {
       state.params = params;
     },
     setQueryPreviewCached(state, queryPreview) {
       Vue.set(state.queriesPreview, getHashFromQueryPreviewItem(queryPreview), queryPreview);
     },
-    setStatus(state, { query, status }) {
-      state.queriesPreview[query].status = status;
+    setStatus(state, { queryPreviewHash, status }) {
+      state.queriesPreview[queryPreviewHash].status = status;
     },
     setSelectedQueryPreview(state, selectedQueryPreview) {
       state.selectedQueryPreview = selectedQueryPreview;
     },
     setConfig,
     mergeConfig,
-    addQueryPreviewInstance(state, query: string) {
-      if (state.queriesPreview[query]) {
-        state.queriesPreview[query].instances += 1;
+    addQueryPreviewInstance(state, queryPreviewHash: string) {
```

Comment:
No need to specify the type in `queryPreviewHash: string`, it's already being done in the types file, remove the `string`.